samples/nrf9160/location/prj.conf

```diff
@@ -56,5 +56,7 @@ CONFIG_DATE_TIME=y
 # Modem JWT
 CONFIG_MODEM_JWT=y
 
-# Multicell location service selection
-CONFIG_MULTICELL_LOCATION_SERVICE_NRF_CLOUD=y
+# Location service selection
+#CONFIG_LOCATION_SERVICE_NRF_CLOUD=y
```

Comment:
Why is this line commented out? It's enabled by default and I think having it in comments like this is a bit confusing. At least my assumption for these kind of lines is that the option is not enabled by default.

Reply:
Jep, maybe it should be `#CONFIG_LOCATION_SERVICE_NRF_CLOUD=n`